tls/test_tls_cert.py

```diff
@@ -281,11 +280,7 @@ def setUp(self):
         tester.TempestaTest.setUp(self)
 
     def test(self):
-        # TODO #1294: Tempesta throws misleading ERROR report on invalid
-        # tls_certificate configuration, just the same as if there is no
-        # certificate file at all, instead of correctly report about
-        # not supported SHA1.
-        self.check_cannot_start("ERROR: configuration parsing error")
+        self.check_cannot_start("1.2.840.10045.4.1 is unsupported")
```

Comment:
May be `Purposes oid 1.2.840.10045.4.1 (ecdsa-with-SHA) is unsupported`?
https://oidref.com/1.2.840.10045.4.1